ui/config/data/general_config.js

```diff
@@ -19,7 +19,7 @@ UserConfig.registerOptions('general', {
       name: {
         en: 'Show developer options',
         de: 'Zeige Entwickleroptionen',
-        fr: 'Afficher les options développeur',
+        fr: 'Afficher les options du développeur',
```

Comment:
If I compare with my android device, it is _Option développeur_. This is not a specific developer (du) it's general.

Reply:
ok.